charts/etcd/templates/etcd-statefulset.yaml

```diff
@@ -46,6 +46,16 @@ spec:
       priorityClassName: {{ .Values.priorityClassName }}
 {{- end }}
       serviceAccountName: {{ .Values.serviceAccountName }}
+{{- if .Values.schedulingConstraints }}
+{{- if .Values.schedulingConstraints.affinity }}
+      affinity:
+{{ toYaml .Values.schedulingConstraints.affinity | indent 8 }}
+{{- end }}
+{{- if .Values.schedulingConstraints.topologySpreadConstraints }}
+      topologySpreadConstraints:
+{{ toYaml .Values.schedulingConstraints.topologySpreadConstraints | indent 6 }}
```

Comment:
Should it not be `indent 8` ?

Reply:
No, it's 6. Because the contents of this field is an array, so the hyphen starts at indentation 6. You can try it by running `helm template`

Reply:
Okay